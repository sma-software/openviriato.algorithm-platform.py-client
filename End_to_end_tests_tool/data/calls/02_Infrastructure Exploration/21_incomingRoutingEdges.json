{
    "restResource": "/nodes/{id}/incoming-routing-edges",
    "httpMethod": "GET",
    "urlParameters": [ "161" ],
    "queryParameters": [
        {
            "key": "EndNodeTrackID",
            "value": "162",
            "optional": true
        }
    ],
    "requestBody": null,
    "CSharpMethodName": "GetIncomingRoutingEdges(routingPoint)",
    "CSharpMethodParameterTypes": [ "SMA.AlgorithmPlatform.AlgorithmInterface.AIDM.RoutingPoint" ],
    "subcategory": "Get Incoming Routing Edges"
}