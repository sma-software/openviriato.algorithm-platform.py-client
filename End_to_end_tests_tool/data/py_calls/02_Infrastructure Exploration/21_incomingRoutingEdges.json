{
  "PythonObjectMapping" : [
    {
      "ParameterName" : "routing_point",
      "ClassName": "RoutingPoint",
      "PythonParameterMapping": {
        "node_id" : "$.urlParameters.[0]",
        "node_track_id": "$.queryParameters.[0].value"
      }
    }
  ]
}