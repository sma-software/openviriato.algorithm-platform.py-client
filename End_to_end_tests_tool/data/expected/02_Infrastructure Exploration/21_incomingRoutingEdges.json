{"RoutingEdges":
  [
    {
      "StartSectionTrackID":885,
      "EndNodeTrackID":162,
      "NodeID":161
    },
    {
      "StartSectionTrackID":886,
      "EndNodeTrackID":162,
      "NodeID":161
    },
    {
      "StartSectionTrackID":887,
      "EndNodeTrackID":162,
      "NodeID":161
    }
  ]
}